### DISCUSSION TITLE
Which column type would you use to auto-create product SKUs?

### DISCUSSION THREAD

**Discussion by distresscode** [nocodb/nocodb#4187] [Q&A] [answered]
Hello all. I'd like to have a SKU number created when I make a new row. Would the "AutoNumber" be appropriate for this? If so, what formula would you use if I wanted the following format:

two letters - (serialized numbers starting from 1000)
example: DC-1001

Thanks!

**Comment by dstala** [comment, marked as answer]
@distresscode You can use 'System field' Id (Go to fields, enable `Show system fields`)
![Screenshot 2022-10-28 at 7 20 30 PM](https://user-images.githubusercontent.com/86527202/198624019-fdaa79ea-ed75-46d8-bbc0-97d14d3a7f94.png)
![Screenshot 2022-10-28 at 7 20 41 PM](https://user-images.githubusercontent.com/86527202/198624054-fe88bc6f-3a92-4680-9190-444ae54f5da9.png)

**Reply by distresscode** [reply]
That worked beautifully. Thank you!
